src/opm/parser/eclipse/EclipseState/AquiferCT.cpp

```diff
@@ -175,4 +175,11 @@ const std::vector<AquiferCT::AQUCT_data>& AquiferCT::data() const {
     return this->m_aquct;
 }
 
+bool AquiferCT::hasAquifer(const int aquID) const {
+    const auto it = std::find_if(this->m_aquct.begin(), this->m_aquct.end(),
+                                 [&aquID](const auto& aqu){ return aqu.aquiferID == aquID; });
+
+    return ( it != this->m_aquct.end() );
```

Comment:
Any reason to not use `std::any_of()` here?
```C++
return std::any_of(this->m_aquct.begin(), this->m_acuct.end(),
                   [acuID](const auto& aqu) { return aqu.aquiferID == aquID; });
```

Reply:
No other reason, I just has not used `std::any_of` before.